test/add-permissions-tests.js

```diff
@@ -48,6 +52,22 @@ describe('serverless-plugin-lambda-account-access', function() {
 
       sinon.assert.calledOnce(stub);
     });
+
+    it('should create defineFunctionProperties on configSchemaHandler', function() {
+      const instance = createTestInstance();
+      expect(instance)
+        .to.have.property('serverless')
+        .that.has.property('configSchemaHandler')
+        .that.has.property('defineFunctionProperties');
+    });
+
+    it('should create defineProvider on configSchemaHandler', function() {
+      const instance = createTestInstance();
+      expect(instance)
+        .to.have.property('serverless')
+        .that.has.property('configSchemaHandler')
+        .that.has.property('defineProvider');
+    });
```

Comment:
`configSchemaHandler` is provided by `serverless`. We do not create it.

Reply:
Yeah ... that unit test was weird, I guess I was just testing the test initialization. Should be better now.